src/Gui/Timeline/TimelineScrollArea.cpp

```diff
@@ -154,7 +154,7 @@ void TimelineScrollArea::wheelEvent( QWheelEvent* event ) {
         }
 
         double hScroll = horizontalScrollBar()->value();
-        double x       = event->position().x();
+        double x       = event->x();
```

Comment:
QWheelEvent::x() is deprecated since 5.15. Consider adding this 

```suggestion
#if QT_VERSION < QT_VERSION_CHECK( 5, 15, 0 )
            double x = event->x();
#else
            double x =event->position().x();
#endif
```

If fine with this, I could add similar code in all wheel events to remove deprecated warnings.

Reply:
I'm fine with this, even if I didn't find this deprecation in the documentation https://doc.qt.io/qt-5/qmouseevent.html
Qt 5.15 is the last qt5 version if i understand well the Qt documentation and Radium is not Qt6 compatible (but this might be done in the future ?)

Reply:
Well, it's because it's a Wheel event, not a Mouse event
Qt doc https://doc.qt.io/qt-5/qwheelevent.html do not mention .x() at all since it's deprecated ( see https://github.com/qt/qtbase/blob/40143c189b7c1bf3c2058b77d00ea5c4e3be8b28/src/gui/kernel/qevent.h#L216 )

Reply:
OK, BTW, I've pushed the suggested change.